### DISCUSSION TITLE
Cant Download videos from some sites and from some sites cable to download but having green background no audio

### DISCUSSION THREAD

**Discussion by codebysumesh** [aclap-dev/video-downloadhelper#863] [Help and Q&A]
### Your question

Please help me to resolve this?

### Addon & CoApp version and details

Issue 1 I am trying to download videos from some  site but can't download videos properly having not audio and green screen could you please help me with these any settings or anything need to change 

 I have firefox premium version   

Issue 2 Also from some sites videos are not fetching

Version 9.0.2.3 (stable), target mozilla, locale en-US.
Companion App installed (v2.0.19)

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

Your problem description is not proper.  Click the link at the top of any page of this forum labelled Wiki.  Look for the chapter "I need help."  Click that & follow the instructions.

As for scrambled results, search this forum with "drm" as your search key.  Make sure your Filter is set to All, not some other value.
